### PR TITLE
Further expand and update the merge and commit process for committers

### PR DESCRIPTION
**Pull request by srowen**
Following up on https://github.com/apache/spark-website/commit/eb0aa14df472cff092b35ea1b894a0d880185561#r31886611 with additional changes.

### CODE REVIEW

File: committers.md [hunk 1]
```diff
@@ -127,13 +127,41 @@ Git history for that code to see who reviewed patches before. You can do this us
...
+
+```
+apache	https://github.com/apache/spark-website.git (fetch)
+apache	https://github.com/apache/spark-website.git (push)
+apache-github	git://github.com/apache/spark-website (fetch)
+apache-github	git://github.com/apache/spark-website (push)
+origin	https://github.com/[your username]/spark-website.git (fetch)
+origin	https://github.com/[your username]/spark-website.git (push)
+upstream	https://github.com/apache/spark-website.git (fetch)
+upstream	https://github.com/apache/spark-website.git (push)
```
Comment (dongjoon-hyun) [inline committers.md]: In this context, these should be `spark.git` instead of `spark-website.git`.

**Comment by dongjoon-hyun** [comment]
Great! Thank you for updating, @srowen .

File: committers.md (hunk 1 above)
Comment (srowen) [reply committers.md]: Oops yeah ran this in the wrong directory

**Review by dongjoon-hyun — APPROVED** [review]
+1, LGTM.

**Pull request closed without merging**